### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HarlemSquirrel**
Add info to install official Slack beta client

### CODE REVIEW

**Comment by AnnJohn** [comment]
@aturkewi hey figured you'd be the best person to look at this PR since you wrote the reading. thanks!

File: README.md [hunk 1]
```diff
@@ -70,6 +70,15 @@ Next, we run the commands to install the app:
 
 For more information on ScudCloud, you can check out their github: https://github.com/raelgc/scudcloud
 
+### Slack for Linux (Beta)
+
+You can try the official beta client, but keep in mind that it is still in the works. There are binary packages for Ubuntu and Fedora that can be found here: http://skillcrush.com/2015/04/09/land-junior-web-developer-job/
```
Comment (aturkewi) [inline README.md]: @HarlemSquirrel can you update this link? I couldn't find the install on there.

**Comment by aturkewi** [comment]
@HarlemSquirrel Looks great. I didn't realize slack had a beta for linux. I made one comment, and as soon as you update then I can merge the PR. 
Thanks!

**Comment by HarlemSquirrel** [comment]
Ah yes, that was definitely the wrong link! Fixed now.

File: README.md (hunk 1 above)
Comment (HarlemSquirrel) [inline README.md]: Yup, that was definitely wrong, sorry.

**Comment by aturkewi** [comment]
@HarlemSquirrel looks good! Thanks for the update.

**Pull request merged**